README.md

```diff
@@ -10,4 +10,6 @@ Please contact us directly for any security issue contact@geokretymap.org
 
 cf. [INSTALL.md](INSTALL.md) and [CONTRIBUTING.md](CONTRIBUTING.md) readme files.
 
-
+# Credits
+![powered by F3](https://cdn.geokrety.org/images/logos/f3.svg)\
+Powered by [Fat-Free Framework](https://fatfreeframework.com)
```

Comment:
I think F3 is introduced with **feature/new-theme** branch, so what about making this PR for **feature/new-theme** branch as target (instead of geokrety:master)

but we dont have credits in new-theme right now @kumy ?